src/omero_zarr/cli.py

```diff
@@ -87,6 +87,20 @@ def _configure(self, parser):
             type=ProxyStringType("Image"),
             help="The Image from which to export Masks.",
         )
+        masks.add_argument(
+            "--group-rois",
+            action="store_true",
+            help="Store each ROI in a separate group.",
+        )
+        masks.add_argument(
+            "--mask-bits",
+            default=str(max(MASK_DTYPE_SIZE.keys())),
+            choices=[str(s) for s in sorted(MASK_DTYPE_SIZE.keys())],
+            help=(
+                "Integer bit size for each mask pixel, use 1 for a binary "
```

Comment:
When would someone use this?

Reply:
If you want a binary image rather than a labelled image, or if you want to flatten all masks. Particularly useful if you use group-rois.